client/src/Edit/elements/ReferenceInput.tsx

```diff
@@ -130,24 +130,16 @@ export default class ReferenceInput extends Component<Props, State> {
 
   fetchItems = opts => {
     const { type, models } = this.props;
-    const [firstModel, ...otherModels] = models;
 
-    const queryString = query({
+    const queryString = stringify({
       ...opts,
       linkable: models.length === 0,
-      models: otherModels && otherModels.length ? models : undefined
+      models: models.length ? models : undefined
     });
 
-    // if reference has only one options, use list instead of search
-    return api
-      .get(
-        `/${type}${
-          models.length === 0 || otherModels.length > 0 ? "" : `/${firstModel}/`
-        }${queryString}`
-      )
-      .then(res => {
-        if (res) this.setState({ items: res.items });
-      });
+    return api.get(`/${type}?${queryString}`).then(res => {
+      if (res) this.setState({ items: res.items });
```

Comment:
What does it mean when res is falsy? What is the not-happy-path here?

Reply:
I can't tell you, I would guess this is just a defensive way of accessing the response.

Reply:
Just noticed, that it was like this beforehand. Smells like errors could be swallowed here.

Reply:
Good point, I'm gonna change that.